tests/baselines/reference/jsDeclarationsGetterSetter.js

```diff
@@ -280,5 +280,5 @@ export class L {
     set x(value: any);
 }
 export class M {
-    set x(value: any);
+    set x(value: boolean);

```

Comment:
This comes from:
```ts
/**
 * @param {number | boolean} v
 */
const mSetter1 = (v) => {}
/**
 * @param {string | boolean} v
 */
const mSetter2 = (v) => {}
export class M {}
Object.defineProperty(M.prototype, "x", {
    set: Math.random() ? mSetter1 : mSetter2
});
```

This feels like an improvement, those setter functions can be combined safely to a single signature with a `boolean` parameter